packages/analytics/src/tune.js

```diff
@@ -23,33 +23,81 @@ const methods = {
 const allMethods = Object.keys(methods).join(',');
 
 /**
- * Take all `Object` and sort them with selected field
+ * Create and replace the id with a unify id that can be used with [sort](#sort)
  *
- * ```json
- * [{
- * }]
- * ```
+ * Créer et remplacer l'identifiant par un identifiant unifié qui peut être utilisé avec [sort](#sort)
  *
- * Script:
+ * ### Example / Exemple
+ *
+ * #### Script / Scénario
  *
  * ```ini
+ * ; Import analytics plugin required to use tune
+ * ; Importation du plugin analytique nécessaire pour utiliser tune
  * [use]
  * plugin = analytics
  *
+ * ; Using "tune" with default settings
+ * ; Utilisation de "tune" avec les paramettre par defaut
  * [tune]
  *
  * ```
  *
- * Output:
+ * #### Input / Entrée
+ *
+ * ```json
+ *  [
+ *      {
+ *          "id": 1,
+ *          "value": 1
+ *      },
+ *      {
+ *          "id": 2,
+ *          "value": 2
+ *      }
+ *  ]
+ * ```
+ *
+ * #### Output / Sortie
  *
  * ```json
- * [
- * ]
+ *  [
+ *      {
+ *          "id": "0000000000000000001.00000000000000000000",
+ *          "value": {
+ *              "id": 1,
+ *              "value": 1,
+ *              "label": "static value"
+ *          }
+ *      },
+ *      {
+ *          "id": "0000000000000000002.00000000000000000000",
+ *          "value": {
+ *              "id": 2,
+ *              "value": 2,
+ *              "label": "static value"
+ *          }
+ *      }
+ *  ]
  * ```
  *
  * @name tune
- * @param {String} [path=id] path to use for the sort key
- * @returns {Object}
+ * @param {String} [path=id]
+ *      <ul><li>path of the element used to create the unified identifier</li></ul>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why not in Markdown?

Doesn't it work using simple Markdown list ?
It would be more readable (at least in source code comments).

Reply:
Because the document generator changes the indents, markdown code comments may not work properly for parameter documentation.